src/libraries/System.Linq/src/System/Linq/SequenceEqual.cs

```diff
@@ -22,33 +22,22 @@ public static bool SequenceEqual<TSource>(this IEnumerable<TSource> first, IEnum
                 ThrowHelper.ThrowArgumentNullException(ExceptionArgument.second);
             }
 
-            if (comparer == null)
```

Comment:
Nit: would have preferred if this got merged as a separate commit.

Reply:
Separated it into its own commit